DOCUMENTATION.md

```diff
@@ -2827,6 +2827,43 @@ for await (const row of sqlResult) {
 }
 ```
 
+The "json-flat" value format means that top level fields of the json object is treated as separate columns. There is also "json"
+value format which we cover in the next section.
+
+### 8.7.7. JSON Data Type
+
+JSON is a first-class SQL type with IMap and Kafka Connector support, usable in most contexts, similarly to any other type.
+
+In order to send a JSON parameter use a `HazelcastJsonValue` object:
```

Comment:
Can't this also be a regular JS object? IMHO, if JS objects work fine, we should use it in code snippets

Reply:
If a user configures a global serializer I think using object won't work. This way it is safer.

Reply:
The ones who configure a global serializer is probably around %0.1 of our users :) 

I would mention JS objects for the vast majority